src/datachain/lib/signal_schema.py

```diff
@@ -534,12 +541,12 @@ def _build_tree(
             for name, val in values.items()
         }
 
-    def get_flat_tree(self) -> Iterator[tuple[list[str], type, bool, int]]:
+    def get_flat_tree(self) -> Iterator[tuple[list[str], DataType, bool, int]]:
```

Comment:
Fix type here: `DataType` should be used because it is returned from `self.tree`. Otherwise we will have mypy warnings.